### PR TITLE
Feature/wallet release

### PR DESCRIPTION
**Pull request by martineehrlich**
CCXT custom command that allows the user to use the release command on the broker to take funds out of channel.

### CODE REVIEW

**Comment by martineehrlich** [comment]
[Create `release funds` command for CCXT (custom command)](https://trello.com/c/jNtoFlF0/689-create-release-funds-command-for-ccxt-custom-command)

File: js/sparkswap.js [hunk 1]
```diff
@@ -265,7 +265,10 @@ module.exports = class sparkswap extends Exchange {
         });
     }
 
-    async release () {
-        throw new ExchangeError ('Not Implemented');
+    async release (symbol) {
+        await this.loadMarkets ();
+        return await this.privatePostV1WalletRelease ({
```
Comment (dannypaz) [inline js/sparkswap.js]: We can remove `await` on this line

**Review by dannypaz — APPROVED** [review]
LGTM @martineehrlich

**Comment by dannypaz** [comment]
[add withdraw to ccxt implementation](https://trello.com/c/3Y8NLDFs/763-add-withdraw-to-ccxt-implementation)

**Pull request merged**